stablehlo/tests/interpret_tuple_and_get_tuple_element.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+func.func @tuple() {
+  %val0 = stablehlo.constant dense<[1.0, 2.0]> : tensor<2xf64>
+  %val00 = stablehlo.constant dense<3> : tensor<i64>
+  %val1 = stablehlo.tuple %val00 : tuple<tensor<i64>>
+  %operand = stablehlo.tuple %val0, %val1 : tuple<tensor<2xf64>, tuple<tensor<i64>>>
+  func.return
+}
+
+// -----
+
+func.func @get_tuple_element() {
+  %val0 = stablehlo.constant dense<[1.0, 2.0]> : tensor<2xf64>
+  %val00 = stablehlo.constant dense<3> : tensor<i64>
```

Comment:
Is this a name we've used in other tests? If not lets use a different name. I thought this was coordinates, like `val00 == tuple[0][0]`  but that isn't the case. How about `val1_input` or `tmp1`, or even `val1_val0`?

Reply:
Updated and used `tmp` as that is the shortest one and easiest to read.